db/seeds.d/100-installation_media.rb

```diff
@@ -10,11 +10,6 @@
       :os_family => "Redhat",
       :path => "http://mirror.centos.org/centos/$major/os/$arch",
```

Comment:
This can be dropped next year when CentOS 7 goes EOL and mirror.centos.org disappears.